cpu/fe310/Kconfig.clk

```
@@ -0,0 +1,52 @@
+# Copyright (c) 2020 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menu "FE310 clock configuration"
```

It would not hurt to make this depend on the CPU family
```suggestion
menu "FE310 clock configuration"
    depends on CPU_FAM_FE310
```